test/stacktraces.jl

```diff
@@ -0,0 +1,80 @@
+# This file is a part of Julia. License is MIT: http://julialang.org/license
+
+# Tests for /base/stacktraces.jl
+
+# Some tests don't currently work for Appveyor 32-bit Windows
+const APPVEYOR_WIN32 = (
+    OS_NAME == :Windows && WORD_SIZE == 32 && get(ENV, "APPVEYOR", "False") == "True"
```

Comment:
that sounds odd. i may need to power on my windows machine to see what's up with backtraces now